core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -463,6 +464,7 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
         });
         siteAnalyticsService.registerPublishExplorationEvent(
           explorationData.explorationId);
+        alertsService.addSuccessMessage('Exploration Published!');
```

Comment:
Let's drop this one. We're planning to do something else on publication, which is to show a congratulatory modal.